code/obj/machinery/computer/card.dm

```diff
@@ -38,6 +38,26 @@
 	icon = 'icons/obj/computerpanel.dmi'
 	icon_state = "id2"
 
+/obj/machinery/computer/card/portable
+  name = "Portable Id Computer"
+  icon_state = "idport"
+
+  New()
+    ..()
+    src.AddComponent(/datum/component/foldable, /obj/item/objBriefcase/hopid)
```

Comment:
Prefer not tying the briefcase's appearance to what is inside it. (i.e. `hopid` is a bad name for the subtype, as it could logically contain anything).

Reply:
how would i change this, so when it unfolds it is the "idport" briefcase? without this it becomes an ID computer